src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -110,7 +110,7 @@ public function call($command, $parameters)
             $this->image->layers()->coalesce();
             foreach ($this->image->layers() as $layer) {
                 $counter++;
-                $this->commandManager->get($command)->execute($layer, $parameters);
+                $this->commandManager->getCommand($command)->execute($layer, $parameters);
```

Comment:
I think also we should use `get` and then the `addCommand` should also be `add`, else all looks fine for me.